packages/main/src/TimePickerBase.ts

```diff
@@ -483,6 +568,28 @@ class TimePickerBase extends UI5Element {
 		e.preventDefault();
 	}
 
+	_hideMobileKeyboard() {
+		this._getInput().readonly = true;
+		setTimeout(() => { this._getInput().readonly = false; }, 0);
```

Comment:
Why do we set this to true, then move it to the end of event loop and change it to false?